### ISSUE TITLE
how to select the best model ?

### ISSUE THREAD

**Issue by rockywind** [TRAILab/CaDDN#59]
Hi, after finishing training the KITTI 3D detection train + val set for 80 Epochs, which one of the trained models would be the best for submitting to the KITTI official?

**Comment by codyreading** [comment]
Since we don't have an additional validation set, I recommend to pick the final model at Epoch 80.

**Comment by rockywind** [comment]
Thanks!
